src/actions/vstudio/vs2005_csproj_user.lua

```diff
@@ -65,7 +66,11 @@
 		end
 	end
 
-
+    function m.localDebuggerWorkingDirectory(cfg)
```

Comment:
En, when I modify this code, I'm in company, and I also know this fix code can not place here... I'm very sorry, Is in a hurry, for this change, I will modify the code again and request pull again,